GANDLF/losses/loss_calculators.py

```diff
@@ -0,0 +1,83 @@
+import torch
+from GANDLF.losses import get_loss
+from abc import ABC, abstractmethod
+from typing import List
+
+
+class AbstractLossCalculator(ABC):
+    def __init__(self, params: dict):
+        super().__init__()
+        self.params = params
+        self._initialize_loss()
+
+    def _initialize_loss(self):
+        self.loss = get_loss(self.params)
+
+    @abstractmethod
+    def __call__(
+        self, prediction: torch.Tensor, target: torch.Tensor, *args
+    ) -> torch.Tensor:
+        pass
+
+
+class LossCalculatorStdnet(AbstractLossCalculator):
+    def __init__(self, params):
+        super().__init__(params)
+        self.l1_loss = get_loss(params)
+        self.kld_loss = get_loss(params)
+        self.mse_loss = get_loss(params)
```

Comment:
How are the different losses triaged in the config?

Reply:
Will be resolved in another PR